dot/telemetry/afg_received.go

```diff
@@ -7,6 +7,10 @@ import "github.com/ChainSafe/gossamer/lib/common"
 
 // AfG ("Al's Finality Gadget") is synonymous with GRANDPA.
 
+var _ Message = (*AfgReceivedPrecommitTM)(nil)
+var _ Message = (*AfgReceivedPrevoteTM)(nil)
+var _ Message = (*AfgReceivedCommitTM)(nil)
```

Comment:
nit up to you, feel free to ignore (FFTI)

```suggestion
var (
	_ Message = (*AfgReceivedPrecommitTM)(nil)
	_ Message = (*AfgReceivedPrevoteTM)(nil)
	_ Message = (*AfgReceivedCommitTM)(nil)
)
```